tests/arcus/ArcusCommunicationPrivateTest.cpp

```diff
@@ -155,18 +163,20 @@ TEST_F(ArcusCommunicationPrivateTest, ReadMeshGroupMessage)
 
     // - Load 'global' settings:
     std::unordered_map<std::string, std::string> raw_settings;
-    loadTestSettings("../tests/test_global_settings.txt", &mesh_message, &raw_settings);
+    loadTestSettings(std::filesystem::path(__FILE__).parent_path().parent_path().append("test_global_settings.txt").string(),
```

Comment:
If your aim with breaking the line into multiple pieces was to improve readability, I suggest to calculate the path in a separate line and not break the function call into pieces.